tests/test_bibtex.py

```diff
@@ -110,11 +110,11 @@ def test_pages(self):
     (r"Victor, Paul {\'e}mile", (r"Paul {\'e}mile", None        , "Victor", None)),
 
     ('Dominique Galouzeau de Villepin',
-      ('Dominique Galouzeau', 'de'            , 'Villepin'             , None)),
```

Comment:
I can add more codes to the putty-ignore list so that the number of lines modified in this file drops down to a few more serious readibility issues.